SaveContext.py

```diff
@@ -1045,7 +1053,10 @@ def get_save_context_addresses(self):
             'double_magic'          : [False, True],
         },
         "Rupee"                     : {'rupees' : None},
-        "Rupee (Treasure Chest Game)" : {'rupees' : None},
+        "Green Rupee (Treasure Chest Game)" : {'rupees' : None},
+        "Blue Rupee (Treasure Chest Game)" : {'rupees' : None},
```

Comment:
These need to be changed from `None` to their actual values, since the values can't be derived from the item names.

Reply:
Done